src/main/java/com/playtika/automation/school/test/framework/action/RegistrationActions.java

```diff
@@ -0,0 +1,48 @@
+package com.playtika.automation.school.test.framework.action;
+
+import com.playtika.automation.school.test.framework.client.RegistrationFeignClient;
+import com.playtika.automation.school.test.framework.pojo.RegistrationRequest;
+import com.playtika.automation.school.test.framework.pojo.responses.RegistrationResponse;
+import lombok.AllArgsConstructor;
+
+import java.util.Random;
+
+@AllArgsConstructor
+public class RegistrationActions {
+
+    private final RegistrationFeignClient registrationFeignClient;
+
+    private static final String MAIL = "mail_";
+    private static final String MAILPOSTFIX = "@yandex.ru";
+    private static final String PASSWORD = "Password_";
+    private static final int RANDOMLOW = 1;
+    private static final int RANDOMHIGH = 100000;
+
+    public RegistrationResponse getRegistrationData(String mail, String password) {
+        RegistrationRequest request = RegistrationRequest.builder()
+                .email(mail)
+                .password(password)
+                .build();
+        return registrationFeignClient.createUser(request);
+    };
+
+    private Integer generateRandomInt() {
```

Comment:
private methods should be placed after public and package-private and protected methods